src/renderers/webgl/index.ts

```diff
@@ -30,6 +31,15 @@ export type NodeDragEvent = { type: 'nodeDrag', x: number, y: number, clientX: n
 export type EdgePointerEvent = { type: 'edgePointer', x: number, y: number, clientX: number, clientY: number, target: Graph.Edge, altKey?: boolean, ctrlKey?: boolean, metaKey?: boolean, shiftKey?: boolean }
 
 
+export type AnnotationPointerEvent = { type: 'annotationPointer', x: number, y: number, clientX: number, clientY: number, target: Graph.Annotation, altKey?: boolean, ctrlKey?: boolean, metaKey?: boolean, shiftKey?: boolean }
+
+
+export type AnnotationDragEvent = { type: 'annotationDrag',  x: number, y: number, clientX: number, clientY: number, annotationX: number, annotationY: number,  target: Graph.Annotation, altKey?: boolean, ctrlKey?: boolean, metaKey?: boolean, shiftKey?: boolean }
+
+
+export type AnnotationResizeEvent = { type: 'annotationResize',  x: number, y: number, dx: number, dy: number,  target: Graph.Annotation, altKey?: boolean, ctrlKey?: boolean, metaKey?: boolean, shiftKey?: boolean }
```

Comment:
comparing this event to others like the viewport drag/wheel event, looks like the convention is to return a new position/zoom, rather than deltas. so maybe here makes more sense to return new width/height, rather than dx/dy?